examples/intraday_examples/lead_time_analysis.py

```diff
@@ -0,0 +1,312 @@
+                                            delta_hours=time_delta)
+        absolute_curve.name = map_name_to_label(absolute_curve.name)
+        df = pd.concat([df, absolute_curve], axis=1)
+    return df
+
+
+def timezone_mapping(area: str) -> str:
+    """This function defines a timezone depending on the area input.
+
+    Args:
+        area: User defined area.
+
+    Returns:
+        str: The timezone displayed as a string.
+    """
+    if area.upper() == "TR":
+        return "trt"
+    elif "IE" in area.upper():
+        return "wet"
+    else:
+        return "cet"
+
+
+def map_name_to_label(curve_name: str):
+    """This function matches one name from the defined dictionary below with an inputted curve name and returns a more readable name.
+
+    Args:
+        curve_name: A curve name, for instance: "pro fr wnd ec00 mwh/h cet min15 f 2023-11-26T00:00:00+01:00"
+
+    Returns:
+        str: A name from the dictionary generated in this function. Can be used as column names in a df."
+    """
+    mapping = dict(spv="Spv production", wnd="Wind production", con="Consumption", rdl="Residual load")
+    return [mapping.get(name, 'Undefined') for name in mapping if name in curve_name][0]
+
+
+def str_to_datetime(time: pd.Timestamp, area: str) -> pd.Timestamp:
+    """This function takes a taimestamp and an area is inputs and assigns a timezone to the timestamp based on the area.
+
+    Args:
+        time: A pandas timestamp.
+        area: user defined area name, such as DE, FL, etc.
+
+    Returns:
+        pd.Timestamp: A pandas timestamp with timezone is returned.
+    """
+    return pd.Timestamp(time, tz=timezone_mapping(area))
+
+
+def create_dataframe(instance_curves_df: dict, absolute_curves_df: dict, area: str,
+                     data_time: pd.Timestamp) -> pd.DataFrame:
+    """This function takes two dataframes as input, instance_curves_df and absolute_curves_df and subtracks each of the baseforecasts'
+    value at the data time timestamp from each value in the corresponding columns in the intraday dataframe.
+    Args:
+        instance_curves_df: A dataframe containing all retrieved data from all the benchmark curves.
+        absolute_curves_df: A dataframe containing all retrieved data from all the intraday curves.
+        area: The user defined area.
+        data_time: The timestamp of the forecasted value.
+
+    Returns:
+        difference_curves: A dataframe containing the difference values between benchmark forecast and intraday forecast. Ready to be plotted.
+    """
+
+    data_time_timezone = str_to_datetime(time=data_time, area=area)
+    difference_curves = absolute_curves_df.sub(instance_curves_df.loc[data_time_timezone]).dropna()
+
+    # Multiply spv and wind production with -1 to align residual load curve with stacked bar plot
+    difference_curves.loc[:, ["Spv production", "Wind production"]] = difference_curves.loc[:,
```

Comment:
I'm not sure, but could you try doing difference_curves.loc[:, ["Spv production", "Wind production"]] *= -1 instead?

Reply:
Looks like it worked